### ISSUE TITLE
`corner_plot` returns a `ValueError` if `istype(truths,np.ndarray)`

### ISSUE THREAD

**Issue by CChapmanbird** [mj-will/nessai#373]
**Describe the bug**
`corner_plot` returns a `ValueError` if an `ndarray` is passed for the `truths` kwarg. A `list` object is fine.

**To Reproduce**
```python
from nessai.plot import corner_plot
from nessai.livepoint import dict_to_live_points
import matplotlib.pyplot as plt
import numpy as np

truth = np.array([
    0.,
    0.,
])

samples = {
    "x0":np.random.normal(size=10000),
    "x1":np.random.normal(size=10000)
}

result = dict_to_live_points(samples)

corner_plot(
    result,
    exclude=["logL","logP","it"],
    truths = truth,
)
``` 

**Expected behaviour**
A corner plot should appear.

**Screenshots**
N/A

**Additional context**
nessai version: 0.11.0
python version: 3.12.1

**Comment by mj-will** [comment]
Thanks for reporting this @CChapmanbird, I'll patch this ASAP and then make a pre-release since the next release might be a few weeks off

**Comment by mj-will** [comment]
@CChapmanbird I've made a pre-release that contains a fix for this (https://github.com/mj-will/nessai/releases/tag/v0.12.0a1), hopefully this works but let me know if it doesn't.

**Comment by mj-will** [comment]
This is now addressed in 0.12.0
